Base/QTApp/qSlicerApplicationHelper.cxx

```diff
@@ -66,6 +66,10 @@
 # include <PythonQtPythonInclude.h>
 #endif
 
+#ifdef _WIN32
+#include <Windows.h> //for SetProcessDPIAware
```

Comment:
We've been building Slicer successfully without this on many computers. Do you think it's related to ITK update?

Reply:
I doubt it is related to ITK update. More likely it is some combination of Qt version, Windows SDK version etc.

Reply:
`_WIN32` -> `Q_CC_MSVC` ? 

Let's integrate this separately

Reply:
Sure. Should I make a new PR? Or just merging the first commit is possible?